app-ingress.yaml

```diff
@@ -3,16 +3,15 @@ kind: Ingress
 metadata:
   name: rocketchat-ingress
 spec:
-  tls:
-  - hosts:
-    - chat.example.com
+   tls:
```

Comment:
Removed the unnecessary blanks (messing up the indentation)

Reply:
Done !